web/src/components/apps/DashboardVersionCard.jsx

```diff
@@ -907,16 +909,28 @@ class DashboardVersionCard extends React.Component {
       return null;
     }
 
+    let latestVersion = downstream?.latestVersion;
```

Comment:
let's rename this to something like `nextAppVersion`? because reading the code otherwise is a bit confusing